nextflow.config

```diff
@@ -203,19 +185,28 @@ profiles {
     test_parameters { includeConfig 'conf/test_parameters.config' }
 }
 
-// Set default registry for Apptainer, Docker, Podman and Singularity independent of -profile
-// Will not be used unless Apptainer / Docker / Podman / Singularity are enabled
-// Set to your registry if you have a mirror of containers
-apptainer.registry   = 'quay.io'
-docker.registry      = 'quay.io'
-podman.registry      = 'quay.io'
-singularity.registry = 'quay.io'
+// Load nf-core custom profiles from different Institutions
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
 // Nextflow plugins
 plugins {
     id 'nf-validation@1.1.4' // Validation of pipeline parameters and creation of an input channel from a sample sheet
 }
 
+// Load nf-core/multiplesequencealign custom profiles from different institutions.
+// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/multiplesequencealign.config" : "/dev/null"
```

Comment:
You can uncomment this line, it's to allow the use of an nf-core/config specific for this piepeline

Reply:
ok